components/kyma-environment-broker/internal/broker/instance_last_operation.go

```diff
@@ -2,34 +2,36 @@ package broker
 
 import (
 	"context"
-	"fmt"
 	"net/http"
 
 	"github.com/kyma-incubator/compass/components/kyma-environment-broker/internal/storage"
 
 	"github.com/pivotal-cf/brokerapi/v7/domain"
 	"github.com/pivotal-cf/brokerapi/v7/domain/apiresponses"
 	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
 )
 
 type LastOperationEndpoint struct {
 	operationStorage storage.Operations
-	dumper           StructDumper
+
+	log logrus.FieldLogger
 }
 
-func NewLastOperation(os storage.Operations, d StructDumper) *LastOperationEndpoint {
+func NewLastOperation(os storage.Operations, log logrus.FieldLogger) *LastOperationEndpoint {
 	return &LastOperationEndpoint{
 		operationStorage: os,
-		dumper:           d,
+		log:              log.WithField("service", "LastOperationEndpoint"),
 	}
 }
 
 // LastOperation fetches last operation state for a service instance
 //   GET /v2/service_instances/{instance_id}/last_operation
 func (b *LastOperationEndpoint) LastOperation(ctx context.Context, instanceID string, details domain.PollDetails) (domain.LastOperation, error) {
+	logger := b.log.WithField("instnaceID", instanceID).WithField("operationID", details.OperationData)
```

Comment:
I've added operationID field only for operations endpoint.  It is possible to add field operation ID but not at the beggining of the method (because we don't know the ID)

Reply:
I've redesigned instance_create to log also with operation ID